lib/hammer_cli_foreman/virtual_machine.rb

```diff
@@ -0,0 +1,78 @@
+
+module HammerCLIForeman
+
+  class VirtualMachine < HammerCLIForeman::Command
+
+    resource :compute_resources
+    command_name 'virtual_machine'
+    desc _("View and manage compute resource's virtual machines")
+
+
+    class AvailableVirtualMachinesCommand < HammerCLIForeman::ListCommand
```

Comment:
I'd actually move this command here:

https://github.com/theforeman/hammer-cli-foreman/blob/f163dd9fc9a232f711621f81a16752624edd5de7/lib/hammer_cli_foreman/compute_resource.rb#L213-L226

since this follows the same pattern.